canonical/lxd-ui
# Use config api for network form help texts and defaults

## Done

- Use config api for network form help texts and defaults
- unify network form key handling

Fixes WD-7391

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check create/edit managed networks and ensure the help texts are displayed correctly
